pkg/virt-controller/watch/util/util.go

```diff
@@ -89,7 +89,8 @@ func PodsUsingPVCs(podInformer cache.SharedIndexInformer, namespace string, pvcN
 		}
 
```

Comment:
Nit: IMO it's cleaner to skip succeeded pods instead of adding a new condition in the `if` statement for each pod volume. 
```go
if pod.Status.Phase == corev1.PodSucceeded {
    continue
} 

for _, volume := range pod.Spec.Volumes {
...
```